doc/source/how-to/vulnerabilities.rst

```diff
@@ -0,0 +1,62 @@
+Vulnerabilities
+===============
+
+.. contents::
+
+Vulnerabilities refer to potential security flaws or weaknesses in PyAnsys packages.
+Here are some examples of common vulnerabilities:
+- Insufficient input validation or insecure handling of file paths can lead to unauthorized access
+- Insecure code and command injection into the app
+- Remote code execution due to deserialization of untrusted code
+- Disclosure of sensitive information due to weak exception handling
+
+Vulnerability sources
+---------------------
+
+- **Vulnerabilities from PyAnsys library code**
+  The maintainers are responsible for deciding whether to address vulnerabilities.
+  The priority of vulnerabilities can be escalated internally if
+  they represent a roadblock for business activity, such as.
+  preventing some deal closures or the delivery of products.
+  The Ansys business units in charge of the package should handle
+  vulnerabilities case by case.
+
+- **Vulnerabilities from external package dependencies**
+  When vulnerabilities exist in external packages used by PyAnsys libraries, such as numpy or matplotlib,
+  Ansys cannot address these vulnerabilities directly. It would be an endless task.
```

Comment:
```suggestion
  Ansys should not address these vulnerabilities directly. Instead, it is best to raise an issue on
the open source repository pointing out the vulnerability and linking the applicable CVE.
```

Reply:
What is a CVE?

Reply:
Ah, guess we didn't point that out. https://www.cve.org/